CHANGELOG_iOS.md

```diff
@@ -1,22 +1,58 @@
 # Changelog
 
+## [1.68.134](https://github.com/brave/brave-browser/releases/tag/v1.68.134)
+
+### Web3
+
+ - Added the ability to import BTC accounts (BIP84) into Brave Wallet. ([#38546](https://github.com/brave/brave-browser/issues/38546))
+ - Added the ability to send BTC on “Send” screen. ([#37958](https://github.com/brave/brave-browser/issues/37958))
+ - Updated default account names to differentiate between mainnet and testnet accounts. ([#38526](https://github.com/brave/brave-browser/issues/38526))
```

Comment:
Specific change to Filecoin so better to add that 
`Updated default account names to differentiate between Filecoin mainnet and testnet accounts.`